### DISCUSSION TITLE
Show Originals for Blog Subscriptions (and maybe other views)

### DISCUSSION THREAD

**Discussion by Echoweaver** [AprilSylph/XKit-Rewritten#337] [Suggestion Box]
My favorite feature of XKit-Rewritten is Show Originals, but if I click through to a blog to catch up, the reblogs come back. Is it at all possible to enable Show Originals for other views that the dashboard? That would make the weirdly hard-to-find Blog Subscriptions view usable.

I see that the tagged view does implement Show Originals, so maybe it's just missing from Blog Subscriptions.

I'd love to see Show Originals anywhere I can get it. If it could be implemented for the blogs opened as a sidebar on the desktop, that would also be delightful.

**Comment by marcustyphoon** [comment]
Re: Show Originals for blogs in the sidebar:

One of the [posts on Staff's WIP blog](https://wip.tumblr.com/post/659970876190621696/is-this-a-venue-for-feature-requests-heres) implies that they might be considering this, but I'm unsure if we should take that post as a "hey good idea" or a "hey good idea, we will definitely do this at some point." 

(Additionally, I assume the Tumblr version would only include actual original posts, while the XKit Rewritten version has the option to show reblogs with contributed content, a feature which I really like. So I could still see value in implementing this anyway.)

Re: Show Originals for Blog Subscriptions: Hey, good idea!

In both cases I feel like enabling Show Originals should be optional. (Both seem like great places to catch up on reblogs from only certain people while using this script to filter your dash.) On the one hand, I try to avoid my natural impulse to add a million settings to everything; on the other hand, I imagine these are relevant to most people who would consider this script in the first place.

**Reply by AprilSylph** [reply]
It seems like Show Originals has enough demand for extra settings for them to be added. We could probably make a few checkbox options for which timelines it activates on.

**Reply by Echoweaver** [reply]
I didn't know that sidebar blog reading was called "Peepr." Now I know. So I guess this is kind of a duplicate for an earlier request. 

For my use case, the vast majority of reblogs are just noise, so I don't want to see them at any time except for specific blogs I have already whitelisted. I want the same settings to be active wherever I read on tumblr, but I appreciate that mine isn't the only use case.

**Comment by marcustyphoon** [comment]
April, is this a feature you plan to work on, specifically plan to leave to the users of the extension (so, me), or don't have a plan either way for at the moment? I have no preference, just want to avoid duplicating work or work never getting done!

**Reply by AprilSylph** [reply]
As a rule, if I'm not assigned to an issue (or there is no open issue), you wouldn't be treading on my feet by working on it. I'll be converting more discussions into issues in future so things are more clear, right now I'm only converting ones for the purpose of self-assignment.

**Reply by marcustyphoon** [reply]
Great, works for me :)
